api/app/models/annotation.rb

```diff
@@ -58,6 +62,13 @@ class Annotation < ApplicationRecord
   belongs_to :collection, optional: true
   has_many :comments, as: :subject, dependent: :destroy, inverse_of: :subject,
                       counter_cache: :comments_count
+  has_one :text, through: :text_section
+  has_one :project, through: :text
+  has_one :annotation_created_event, -> { where event_type: Event::TEXT_ANNOTATED },
```

Comment:
Ensure that there is a spec for this behavior:
Annotation enqueues a Event::TEXT_ANNOTATED event on creation

Reply:
✔